rest/src/main/java/org/n52/io/extension/RenderingHintsExtension.java

```diff
@@ -74,7 +74,11 @@ public void addExtraMetadataFieldNames(DatasetOutput output) {
     }
 
     private boolean hasRenderingHints(DatasetOutput output) {
-        return hasSeriesConfiguration(output) || hasPhenomenonConfiguration(output);
+        return hasDatasetParameters(output) && (hasSeriesConfiguration(output) || hasPhenomenonConfiguration(output));
+    }
+
+    private boolean hasDatasetParameters(DatasetOutput output) {
+        return output.getDatasetParameters() != null;
```

Comment:
I don't see serialization flags making `setValue()` unnecessary ... What do you think of keep the actual value in a serialization wrapper type (which is being set within `setValue()`). The wrapper can store the value if it shall be serialized or not. An appropriate overloaded getter could be added then for those cases where value is needed even if not marked for serialization. 

I'll try to come up with a first draft.